css/categorysite.css

```
@@ -27,11 +32,93 @@ content {
 	overflow: hidden;
 }
 
+.matkassar {
+	font-weight: 900;
+	color: white;
+	text-transform: uppercase;
+	margin: 0;
+	position: absolute;
+	top: 50%;
+	left: 50%;
+	font-size: 2rem;
+}
+
+.veg {
+	font-family: 'Roboto', sans-serif;
+	text-transform: uppercase;
+	font-weight: 900;
+	margin: 0;
+	position: absolute;
+	left: 50%;
+	font-size: 2rem;
+	transform: translate(-50%, -50%);
+}
+
+.flex {
+	font-family: 'Roboto', sans-serif;
+	text-transform: uppercase;
+	font-weight: 900;
+	margin: 0;
+	position: absolute;
+	left: 50%;
+	font-size: 2rem;
+	transform: translate(-50%, -50%);
+}
+
+.gluten {
+	font-family: 'Roboto', sans-serif;
+	text-transform: uppercase;
+	font-weight: 900;
+	margin: 0;
+	position: absolute;
+	left: 50%;
+	font-size: 2rem;
+	transform: translate(-50%, -50%);
+}
+
+.morgon {
+	font-family: 'Roboto', sans-serif;
+	text-transform: uppercase;
+	font-weight: 900;
+	margin: 0;
+	position: absolute;
+	left: 50%;
+	font-size: 2rem;
+	transform: translate(-50%, -50%);
+}
+
 .content img {
 	width: 100%;
 	flex-shrink: 1;
 	flex-grow: 1;
 }
 
+.button1{
+	flex-direction: row;
+	justify-content: space-around;
+    background-color: #13693A;
+    border: none;
+    border-radius: 5px;
+    color: white;
+    padding: 5px 16px;
+    text-decoration: none;
+    font-size: 16px;
+    margin:10px;
+    position: center;
+}
+
+.button2 {
+	flex-direction: row;
+	justify-content: space-around;
+    background-color: #13693A;
+    border: none;
+    border-radius: 5px;
+    color: white;
+    padding: 5px 16px;
+    text-decoration: none;
+    font-size: 16px;
+    margin:10px;
+    position: center;
+}
 
 
```

Samt lägger in även css koden från exemplet:
```suggestion
/* Container holding the image and the text */
.content-section-4 {
  position: relative;
  text-align: center;
  font-size: 34px;
}

/* Bottom left text */
.bottom-left {
  position: absolute;
  bottom: 8px;
  left: 16px;
}

/* Top left text */
.top-left {
  position: absolute;
  top: 8px;
  left: 16px;
}

/* Top right text */
.top-right {
  position: absolute;
  top: 8px;
  right: 16px;
}

/* Bottom right text */
.bottom-right {
  position: absolute;
  bottom: 8px;
  right: 16px;
}

/* Centered text */
.centered {
  position: absolute;
  top: 50%;
  left: 50%;
  transform: translate(-50%, -50%);
}
```